Makefile

```diff
@@ -12,6 +12,15 @@ UNAME := $(shell uname)
 export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
 
+define build-settings-doc
+	PYTHONPATH=./$(1) settings-doc generate \
+	-f markdown -m $(2) \
+	-u docs/source/technical_design/environment_settings/$(3).md \
+	--heading-offset 1 \
+	--between "<!-- generated env. vars. start -->" "<!-- generated env. vars. end -->"
+endef
```

Comment:
I don't really like that settings-doc updates files that are in git and you have to run `git restore` after running `make docs` and make sure you don't accidently commit those files when working on documentation.

I would prefer to have some like `boefje.md.in` in git that is used to generate a `boefje.md` that is .gitignore.

Reply:
I'll look into it

Reply:
@dekkers I've rewritten the `make docs` command such that it generates files from scratch through jinja templates. What do you think?